src/overlays/actors/ovl_En_Bomjima/z_en_bomjima.h

```diff
@@ -26,36 +26,47 @@ typedef struct EnBomjima {
     /* 0x296 */ UNK_TYPE1 unk296[4];
     /* 0x29A */ s16 unk_29A;
     /* 0x29C */ UNK_TYPE1 unk29C[4];
-    /* 0x2A0 */ s16 unk_2A0;
+    /* 0x2A0 */ s16 action;
     /* 0x2A2 */ s16 unk_2A2;
     /* 0x2A4 */ Vec3f unk_2A4;
     /* 0x2B0 */ Vec3f unk_2B0;
     /* 0x2BC */ s16 unk_2BC;
     /* 0x2BE */ s16 unk_2BE;
     /* 0x2C0 */ s16 unk_2C0;
     /* 0x2C2 */ s16 unk_2C2;
-    /* 0x2C4 */ s16 unk_2C4;
+    /* 0x2C4 */ s16 cutsceneTimer;
     /* 0x2C6 */ s16 unk_2C6;
     /* 0x2C8 */ s16 unk_2C8;
     /* 0x2CA */ s16 unk_2CA;
-    /* 0x2CC */ f32 unk_2CC;
+    /* 0x2CC */ f32 animLastFrame;
     /* 0x2D0 */ f32 unk_2D0;
-    /* 0x2D4 */ s16 unk_2D4[2];
+    /* 0x2D4 */ s16 cutscene[2];
```

Comment:
Thoughts on calling the array `cutscenes` since array with two elements instead of cutscene?

Reply:
i don't think so, this is how i read it:

cutscene[1] refering to cutscene number one, since each array element can only hold one cutscene.

Reply:
i take that back